packages/story-editor/src/components/videoTrim/trimmerComponents.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Slider from './slider';
+
+export const Menu = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: black;
```

Comment:
We have the `theme.colors.standard.black` color available but actually the WorkSpace background is `theme.colors.bg.primary` instead, I think. Currently the color differs slightly between the menu bg and the general bg (arrow pointing to the border between the two areas, might need to zoom and and set the screen brighter to see this):

<img width="126" alt="Screenshot 2021-09-22 at 13 51 16" src="https://user-images.githubusercontent.com/3294597/134331147-dce99e1c-3eb5-4b15-a805-efceb5e310fc.png">

Reply:
I swear that this background was black and not `theme.colors.bg.primary` in an earlier design in Figma, but it isn't now, so I'll update.

Reply:
Done in 6908ca2085.